wdqs/0.3.0/.travis/build-deploy.sh

```diff
@@ -2,3 +2,7 @@
 #Oneline for full directory name see: https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 docker build "$DIR/../" -t wikibase/wdqs:0.3.0 -t wikibase/wdqs:latest
+
+if [ "$BRANCH" == "master" ]
+    then docker push wikibase/wdqs:0.3.0 && docker push wikibase/wdqs:latest
```

Comment:
It might be nice to have these on multiple lines rather than &&ed together for readability.

Reply:
sure, the && means if the first fails then we don't try to do the second one we just error out. Is that good and should be kept (but broken over two lines)

Reply:
I don't think we should worry too much if one fails and another succeeds.

Although we should probably somehow mark the build as failed if one of the pushes fails?